app/assets/javascripts/comments.js

```diff
@@ -200,6 +200,19 @@ $(() => {
     const splat = content.split(' ');
     const caretPos = $tgt[0].selectionStart;
     const [currentWord, posInWord] = currentCaretSequence(splat, caretPos);
+    const postId = $tgt.data('post');
+    const characterCount = document.getElementById("character_count_"+postId);
+    const createThreadButton = document.getElementById("create_thread_button_"+postId);
+
+    if (caretPos<15 || caretPos>500){
```

Comment:
Spaces around operators, parens, and braces:

```suggestion
    if (caretPos < 15 || caretPos > 500) {
```